### ISSUE TITLE
All leg slot items showing "red" - inherentPool

### ISSUE THREAD

**Issue by Bigginz87** [aeon0/d4lf#249]
Leg slot is not matching whitelist. I am only using imported profiles from maxroll. Every other slot is recognized properly by the whitelist. I determined this is a result of   `inherentPool` affix which is no longer on pants, only boots. Maxroll import profile will set the max/minCount to 1/1 by default. It should not generate this line at all.  
```inherentPool:
    - count:
      - {name: while_injured_your_potion_also_grants_maximum_life_as_barrier}
      maxCount: 1
      minCount: 1```

**Comment by chrisoro** [comment]
What maxroll link are you using?

**Comment by Bigginz87** [comment]
I've used multiple, Maxroll still lists the inherent affix on their planner, so they all do this. I think the fix is to just make the default state of 'minCount: 0' or removed all together. I just removed it from all of my profiles and legs are filtering properly now.

**Comment by chrisoro** [comment]
I'll disable the inherent part for maxroll for now

**Comment by chrisoro** [comment]
Fixed in 5.2.0
